src/stylesheets/components/datepicker.scss

```diff
@@ -0,0 +1,84 @@
+@import '../variables.scss';
+
+.datepicker-container {
+  position: fixed;
+  top: 10px;
+  left: 10px;
+  right: 10px;
+  background: #222231;
```

Comment:
For most of the values in these files, we can declare and use variables from `variables.scss`